src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs

```diff
@@ -94,8 +94,9 @@ internal static RuntimeGraph GetRuntimeGraph(IEnumerable targetFrameworks)
         {
             var tfi = new TargetFrameworkInformation
             {
-                FrameworkName = NuGetFramework.Parse(targetFrameworkInfo.TargetFrameworkMoniker)
+                FrameworkName = GetTargetFramework(targetFrameworkInfo.Properties)
             };
+            tfi.TargetAlias = GetPropertyValueOrNull(targetFrameworkInfo.Properties, ProjectBuildProperties.TargetFramework) ?? tfi.FrameworkName.GetShortFolderName();
```

Comment:
I don't we want tfi.FrameworkName.GetShortFolderName(). 

I'd expect this to never fail.

Reply:
This was because of my misunderstanding that we always need an alias. Changing in next commit.